.gitignore

```diff
@@ -3,3 +3,5 @@ cache
 docs
 html/.DS_Store
 .Rproj.user
+full_code_src
+devel/
```

Comment:
you must not ignore devel. I understand that you don't want to commit things in here. It will be pushed by travis. If you ignore it the autobuild won't work. If I force push devel there it will end up in the repo regardless of the gitignore...

Reply:
OK :+1: